src/contrib/cluster/Akka.Cluster.Tools/Client/ClusterClientSettings.cs

```diff
@@ -19,7 +19,12 @@ public sealed class ClusterClientSettings
     {
         public static ClusterClientSettings Create(ActorSystem system)
         {
+            system.Settings.InjectTopLevelFallback(ClusterClientReceptionist.DefaultConfig());
```

Comment:
This could cause setting the same fallback config multiple times, since there are no restrictions for using `ClusterClientSettings.Create`. It would be better to nest it inside cluster client extension itself (it's always instantiated once).